.github/workflows/ci.yml

```diff
@@ -17,23 +15,34 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          # - macos-latest
-          # - windows-latest
+          - windows-latest
         node:
-          # - 10
           - 12
           - 14
           - 16
 
     steps:
       - name: Checkout source
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v2
 
       - name: Install Node ${{ matrix.node }}
-        uses: actions/setup-node@v2.4.1
+        uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node }}
 
+      - name: Get npm cache directory
+        id: npm-cache-dir
+        run: |
+          echo "::set-output name=dir::$(npm config get cache)"
```

Comment:
This allows this cache to be used cross-platform. Workflow snippet pulled from https://github.com/actions/cache/blob/main/examples.md#using-multiple-systems-and-npm-config